DigitalLearningSolutions.Data.Tests/Services/DiagnosticAssessmentServiceTests.cs

```diff
@@ -398,7 +398,7 @@ public void Get_diagnostic_content_should_not_return_tutorials_where_diagAssess_
         }
 
         [Test]
-        public void Get_diagnostic_content_should_not_return_tutorials_where_archived_date_is_null()
+        public void Get_diagnostic_content_should_not_return_archived_tutorials()
```

Comment:
Might have small merge conflict with @kwunkiuma

Reply:
This didn't turn out be too bad, but thanks for the heads up 🙂